### ISSUE TITLE
Accessibility Assessment

### ISSUE THREAD

**Issue by johnnycase** [projectcontour/contour#3835]
Please run an accessiblity assessment on the project contour docs site and create updates on this issue with any changes needed. This can be done using the Lighthouse tool that is built into the dev tools of chromium browsers.

**Comment by jonasrosland** [comment]
Hi @johnnycase , just wanted to follow up and see what we can do to help you with this? Happy to talk on Slack as well.

**Comment by johnnycase** [comment]
Hey @jonasrosland I do have a question about the scope of the work. Do we need to check accessibility on the whole project contour site or only on the docs portion of the site?

**Comment by johnnycase** [comment]
Accessibility Issues to work:

- Page: https://projectcontour.io/docs/main/
- Link text does not have a sufficient contrast ratio
- Use color #007db1 to fix the color contrast

**Comment by pnbrown** [comment]
It would be good for us to tick off the boxes above that haven't yet been ticked as well as running another accessibility test on the site.
